Microsoft.HealthVault/Connection/SessionCredentialClientBase.cs

```diff
@@ -59,6 +59,8 @@ private SessionCredential GetSessionCredential(HealthServiceResponseData respons
             }
 
             SessionCredential sessionCredential = this.GetAuthenticationToken(responseData.InfoNavigator);
+            // Based on access token default lifetime: https://docs.microsoft.com/en-us/azure/active-directory/active-directory-configurable-token-lifetimes
```

Comment:
... I remember 4 hrs too ..  may be depends on environment

---
In reply to: [115836898](https://github.com/Microsoft/healthvault-dotnetstandard-sdk/pull/27#discussion_r115836898) [](ancestors = 115836898,115809402,115806561,115805648,115798063)